src/app-layer-htp.h

```diff
@@ -257,6 +257,7 @@ typedef struct HtpState_ {
     uint16_t events;
     uint16_t htp_messages_offset; /**< offset into conn->messages list */
     uint32_t file_track_id;             /**< used to assign file track ids to files */
+    uint32_t file_range_id;             /**< used to assign track ids to range file */
```

Comment:
nit: Suggest changing "track ids" to "ids"

Reply:
ok